geode-core/src/main/java/org/apache/geode/internal/cache/wan/GatewaySenderAttributes.java

```diff
@@ -88,6 +88,20 @@
   public boolean enforceThreadsConnectSameReceiver =
       GatewaySender.DEFAULT_ENFORCE_THREADS_CONNECT_SAME_RECEIVER;
 
+  // Added due to "alter gateway-sender" command
+
+  public boolean modifyAlertThreshold = false;
```

Comment:
public mutable fields are not good. I guess this class has other such fields so I'll just skip over this class. If it was me, I'd completely rewrite this class to get make all fields private.

Reply:
+1